programs/vote/src/vote_state/handler.rs

```diff
@@ -872,33 +928,28 @@ impl VoteStateHandler {
         }
     }
 
+    #[cfg(test)]
+    pub fn new_v4(vote_state: VoteStateV4) -> Self {
+        Self {
+            target_state: TargetVoteState::V4(vote_state),
+        }
+    }
+
     #[cfg(test)]
     pub fn default_v3() -> Self {
         Self::new_v3(VoteStateV3::default())
     }
 
     #[cfg(test)]
-    pub fn epoch_credits(&self) -> &Vec<(Epoch, u64, u64)> {
-        match &self.target_state {
-            TargetVoteState::V3(v3) => &v3.epoch_credits,
-        }
+    pub fn default_v4() -> Self {
+        Self::new_v4(VoteStateV4::default())
     }
 
     #[cfg(test)]
-    pub fn nth_recent_lockout(&self, position: usize) -> Option<&Lockout> {
+    pub fn unwrap_v4(&self) -> &VoteStateV4 {
```

Comment:
nit: I think it's more idiomatic to consume `self` instead of `&self` for an `unwrap` style method